identity-service/latest/config/index.md

```diff
@@ -57,6 +52,36 @@ To set the realm file during deployment:
         --namespace $DESIREDNAMESPACE
     ```
 
+## Run Identity Service with Process Services
+
+You can run the Identity Service with Process Services. If you are using Identity Service 1.8.x and above and Process Services 2.3.x and above, you must configure both applications correctly for the logout functionality in Process Services to function.
+
+> **Note:** If you do not configure the Identity Service and Process Services correctly, you will recieve an error when you try and logout using Process Services.
+
+To run the Identity Service with Process Services:
+
+1. In your Identity Service installation navigate to `<location>\alfresco-identity-service-1.8.0\alfresco-identity-service-1.8.0\standalone\configuration` and open `standalone.xml`.
+
+2. Edit the `spi` elements section to include:
+
+    ```xml
+    <spi name="login-protocol">
+        <provider name="openid-connect" enabled="true">
+            <properties>
+              <property name="legacy-logout-redirect-uri"   value="true"/>
+            </properties>
+          </provider>    
+    </spi>
+    ```
+
+3. Save the file and restart your system.
```

Comment:
Because we're dealing with two different systems, specifying which system should be restarted removes some possible ambiguity.

Reply:
Changed.
 3. Save the file and restart the Identity Service.